docs/integrations/sources/pipedrive.md

```diff
@@ -111,7 +111,8 @@ The Pipedrive connector will gracefully handle rate limits. For more information
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                        |
-| :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------- |
+|:--------|:-----------|:---------------------------------------------------------|:---------------------------------------------------------------|
+| 1.0.0   | 2023-06-29 | [27832](https://github.com/airbytehq/airbyte/pull/27832) | Update `Products` schema                                       |
```

Comment:
Please explain why we are updating the schema.  What field was dropped and why?

Reply:
Similarly, this PRs title and description can explain the change better

Reply:
During my work on connector health report, i've mentioned that there is no `followers_count` field in response.
After some googling i've found [an announcement ](https://devcommunity.pipedrive.com/t/removal-of-followers-count-response-field-from-products-endpoints-and-webhook/6135) in PipeDrive Changelog.

Would you like me to rename changes to """ Remove  `followers_count` field from `Products` stream""" ?

Reply:
yes please!